ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -3910,8 +3910,8 @@ static bool ggml_backend_sycl_cpy_tensor_async(ggml_backend_t backend,
     bool is_cpy_supported                = dst->buffer->buft == ggml_backend_sycl_buffer_type(sycl_ctx->device) &&
                             ggml_backend_buffer_is_sycl(src->buffer);
     GGML_SYCL_DEBUG("[SYCL] call %s", __func__);
-    debug_print_tensor(": dst=", dst);
-    debug_print_tensor(" src=", src);
+    GGML_SYCL_DEBUG("%s", debug_get_tensor_str(": dst=", dst).c_str());
+    GGML_SYCL_DEBUG("%s", debug_get_tensor_str(" src=", src).c_str());
```

Comment:
These should also have their `=` removed then

Reply:
Removed as well.